app/products/playbooks/screens/playbook_run/checklist/checklist_item/checklist_item_bottom_sheet.tsx

```diff
@@ -0,0 +1,251 @@
+        ...typography('Body', 100, 'Regular'),
+        color: changeOpacity(theme.centerChannelColor, 0.72),
+    },
+    actionButtonsContainer: {
+        flexDirection: 'row',
+        gap: 12,
+        height: ACTION_BUTTON_HEIGHT,
+    },
+    taskDetailsContainer: {
+        gap: 8,
+    },
+    flex: {
+        flex: 1,
+    },
+}));
+
+type Props = {
+    item: PlaybookChecklistItemModel | PlaybookChecklistItem;
+    assignee?: UserModel;
+    onCheck: () => void;
+    onSkip: () => void;
+    onRunCommand: () => void;
+    teammateNameDisplay: string;
+};
+
+const ChecklistItemBottomSheet = ({
+    item,
+    assignee,
+    onCheck,
+    onSkip,
+    onRunCommand,
+    teammateNameDisplay,
+}: Props) => {
+    const theme = useTheme();
+    const styles = getStyleSheet(theme);
+    const intl = useIntl();
+    const isTablet = useIsTablet();
+
+    const dueDate = 'dueDate' in item ? item.dueDate : item.due_date;
+    const isChecked = item.state === 'closed';
+    const isSkipped = item.state === 'skipped';
+    const isCommandRun = Boolean('commandLastRun' in item ? item.commandLastRun : item.command_last_run);
+
+    const handleCheck = useCallback(() => {
+        onCheck?.();
+        dismissBottomSheet();
+    }, [onCheck]);
+
+    const handleSkip = useCallback(() => {
+        onSkip?.();
+        dismissBottomSheet();
+    }, [onSkip]);
+
+    const handleRunCommand = useCallback(() => {
+        onRunCommand?.();
+        dismissBottomSheet();
+    }, [onRunCommand]);
+
+    const renderActionButtons = () => (
+        <View style={styles.actionButtonsContainer}>
+            <OptionBox
+                iconName='check'
+                activeText={intl.formatMessage(messages.check)}
+                text={intl.formatMessage(messages.check)}
+                onPress={handleCheck}
+                isActive={isChecked}
+                testID='checklist_item.check_button'
+            />
+            <OptionBox
+                iconName='close'
+                activeText={intl.formatMessage(messages.skipped)}
+                text={intl.formatMessage(messages.skip)}
+                onPress={handleSkip}
+                isActive={isSkipped}
+                testID='checklist_item.skip_button'
+            />
+            <OptionBox
+                iconName='slash-forward'
+                activeText={intl.formatMessage(messages.rerunCommand)}
+                text={intl.formatMessage(messages.runCommand)}
+                onPress={handleRunCommand}
+                isActive={isCommandRun}
+                testID='checklist_item.run_command_button'
+            />
+        </View>
+    );
+
+    const getAssigneeInfo: () => ComponentProps<typeof OptionItem>['info'] = () => {
+        if (!assignee) {
+            return intl.formatMessage(messages.none);
+        }
+        return {
+            user: assignee,
+            onPress: onUserChipPress,
+            teammateNameDisplay,
+            location: 'PlaybookRun',
```

Comment:
Shouldn't this be a constant from Screens?

Reply:
`location` has already a defined typescript type, so it only accepts one of those constants, not any arbitrary string.